packages/compiler-sfc/__tests__/compileScript/__snapshots__/defineModel.spec.ts.snap

```diff
@@ -123,6 +123,26 @@ return { modelValue }
 })"
 `;
 
+exports[`defineModel() > w/ ReadonlyArray, ReadonlySet, ReadonlyMap and UnknownType 1`] = `
+"import { useModel as _useModel, defineComponent as _defineComponent } from 'vue'
+
+export default /*#__PURE__*/_defineComponent({
+  props: {
+    "modelValue": { type: [String, ReadonlyArray, ReadonlySet, ReadonlyMap], skipCheck: true },
```

Comment:
This doesn't make sense. `ReadonlyArray` is a TS type, not a built-in JavaScript object at runtime.

Reply:
Thank you!
Okay, can I move the `ReadonlyArray`, `ReadonlySet`, `ReadonlyMap` to the next case and return it as an object?